examples/existing-resources/main.tf

```diff
@@ -9,54 +9,25 @@ module "resource_group" {
   existing_resource_group_name = var.resource_group
 }
 
-##############################################################################
-# Create Key Protect instance outside of terraform-ibm-key-protect-all-inclusive module
-##############################################################################
-module "existing_key_protect" {
-  source            = "git::https://github.com/terraform-ibm-modules/terraform-ibm-key-protect.git?ref=v2.4.1"
-  resource_group_id = module.resource_group.resource_group_id
-  region            = var.region
-  tags              = var.resource_tags
-  key_protect_name  = "${var.prefix}-kp"
-}
-
-##############################################################################
-# Create Key Ring outside of terraform-ibm-key-protect-all-inclusive module
-##############################################################################
-
-locals {
-  key_ring_id = "${var.prefix}-existing-key-ring"
-}
-
-module "existing_key_ring" {
-  source      = "git::https://github.com/terraform-ibm-modules/terraform-ibm-key-protect-key-ring.git?ref=v2.3.1"
-  instance_id = module.existing_key_protect.key_protect_guid
-  key_ring_id = local.key_ring_id
-}
-
 ##############################################################################
 # Key Protect All Inclusive
 ##############################################################################
 
 module "key_protect_all_inclusive" {
-  depends_on                         = [module.existing_key_ring]
-  source                             = "../.."
-  resource_group_id                  = module.resource_group.resource_group_id
-  region                             = var.region
-  resource_tags                      = var.resource_tags
-  access_tags                        = var.access_tags
-  create_key_protect_instance        = false
-  existing_key_protect_instance_guid = module.existing_key_protect.key_protect_guid
-  existing_key_map = {
-    # create a new key in an existing key ring
-    (local.key_ring_id) = [
-      "test-key"
-    ]
-  }
-  key_map = {
-    # "ocp" key ring will be created with a key called "ocp-cluster-1-key"
-    "ocp" = [
-      "ocp-cluster-1-key"
-    ]
-  }
+  source                      = "../.."
+  resource_group_id           = module.resource_group.resource_group_id
+  region                      = var.region
+  create_key_protect_instance = false
+  existing_kms_instance_guid  = var.existing_kms_instance_guid
+  keys = [
+    {
+      key_ring_name     = "default"
+      existing_key_ring = true
+      keys = [
+        {
+          key_name = "test-key"
+        }
+      ]
+    }
+  ]
```

Comment:
you need to update `examples/existing-resources/README.md` with the changes you made